packages/trader/src/Stores/Modules/SmartChart/Constants/barriers.js

```diff
@@ -1,19 +1,23 @@
 export const CONTRACT_SHADES = {
+    ACCU: 'NONE_DOUBLE',
```

Comment:
This file is also duplicated, we need to remove it in future.

Reply:
@matin-deriv sure, we will👍🏻

Reply:
@kate-deriv has done it👍🏻